docker-app/qfieldcloud/core/serializers.py

```diff
@@ -26,7 +26,10 @@
 
 
 def get_avatar_url(user: User, request: Request | None = None) -> str | None:
-    if not user.useraccount.avatar:
+    if user.type == user.Type.TEAM:
+        return None
+
+    if not hasattr(user, "useraccount") or not user.useraccount.avatar:
```

Comment:
```suggestion
    if not user.useraccount.avatar:
```
I would keep it like this, if we have a situation where there is no useraccount, we better get an error rather than silently hiding it.

Reply:
Thanks for the suggestion! Yes, better to raise an error early than to hide it!